tests/integration/targets/eos_vrf_global/tests/common/deleted.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- ansible.builtin.debug:
+    msg: Start eos_vrf_global deleted integration tests connection={{ ansible_connection}}
+
+- ansible.builtin.include_tasks: _remove_config.yaml
+
+- ansible.builtin.include_tasks: _populate.yaml
+
+- block:
+    - name: Delete given vrf configuration
```

Comment:
can we have test for where we delete specific vrf instance ? or that is not supported with `delete`  and achieved through `replaced` ?

Reply:
no we can do that using deleted as well!